core/domain/user_services.py

```diff
@@ -34,11 +34,12 @@
 class UserSettings(object):
     """Value object representing a user's settings."""
     def __init__(
-            self, user_id, email, username=None, last_agreed_to_terms=None,
+            self, user_id, email, created_on=None, username=None,  last_agreed_to_terms=None, 
```

Comment:
Why do  you need any of these changes (created_on, spaces around last_agreed_to_terms)?